src/Stryker.Core/Stryker.Core/Options/InputDefinition.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Stryker.Core.Options
+{
+    public interface IInputDefinition
```

Comment:
To be honest, the suffix `Definition` does not really add much semantic value, it's a bit like "Information" suffix - can mean anything. Why not just `Input` and `IInput`? Looks like you don't have these classes yet in the codebase.

Reply:
We chose InputDefinition because the class (and objects) contain all kinds of info and validation for an option. It also contains the actual input. But the class is not the input itself. 

But yeah I get your point. And we also haven't named all implementations "...InputDefinition" but just "...Input". So yeah the Definition doesn't add much. Ill rename it